snapshot.ts

```diff
@@ -94,13 +94,17 @@ export class SnapshotExporter {
     const hash = crypto
       .createHash('sha256')
       .setEncoding('hex')
-
+    
     await pipeline(
       tar.c(
         {
           gzip: true,
           noPax: true,
           portable: true,
+          // NOTE: this flag is necessary because the tar would be function
+          //       of time and this would make if more difficult to test
+          // @ts-ignore
+          noMtime: true 
```

Comment:
Isn't this the line that @nazrhom made me remove yesterday because I was getting an error running `npm run`?

Reply:
Yes, `npm run` was failing because the typescript types does not include the overloaded version of `tar.c` that we are using: that's why I added the `ts-ignore` annotation.


If we don't set noMtime the tests will faile (`npm test`)

Reply:
Great :muscle: